woocommerce/woocommerce
# Use SelectTree component in Category field

### All Submissions:






### Changes proposed in this Pull Request:



Use SelectTree component in Category field

Closes #37043 .



### How to test the changes in this Pull Request:




1. Open the new product manager.
2. Go to Category field.
3. Test, using only keyboard (related keys: up, down, left, right, tab, home, end, tab), selecting categories and creating a new category
4. Test the same usage with mouse
5. Compare the behavior with the previous implementation and see if any regressions are found



### Other information:




### FOR PR REVIEWER ONLY:
